paws_collector.js

```diff
@@ -14,13 +14,64 @@ const async = require('async');
 const debug = require('debug')('index');
 const AWS = require('aws-sdk');
 const fs = require('fs');
+const moment = require('moment');
 
 const AlAwsCollector = require('@alertlogic/al-aws-collector-js').AlAwsCollector;
 const m_packageJson = require('./package.json');
 
 const CREDS_FILE_PATH = '/tmp/paws_creds.json';
 var PAWS_DECRYPTED_CREDS = null;
 
+function calcNextCollectionInterval(strategy, curUntil, pollInterval) {
```

Comment:
Maybe put this in a separate file? or make it a ccalss method.

Though my preferecen is to make this a separate file and export it.

Reply:
+1 to move it into a separate file

Reply:
done